src/components/new-members/index.js

```diff
@@ -16,19 +17,18 @@ const renderNewUserCard = (newMember) => {
   );
 };
 
-const renderNewUser = (newMember, isSuperUser) => {
+const renderNewUser = (newMember, isSuperUser, handleNewMemberDetailsPage) => {
   if (isSuperUser) {
     return (
-      <Link
-        prefetch={false}
-        href={{
-          pathname: '/[id]',
-        }}
-        as={`/${newMember.username}`}
+      <div
+        role="button"
+        tabIndex={-1}
         key={newMember.username}
+        onClick={() => handleNewMemberDetailsPage(newMember.username)}
+        aria-hidden="true"
```

Comment:
role="button": - it is helpful for screen readers to identify it as a button but it will provide no button functionality.
tabIndex={-1} - tabIndex allows developers to make HTML elements focusable and a negative value means that element is not reachable via sequential keyboard navigation.
aria-hidden="true" - Adding aria-hidden="true" to an element removes that element and all of its children from the accessibility tree. This can improve the experience for assistive technology.